.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
this .idea/ folder is created by the IDE, the folder could be added to the .gitignore to avoid pushing it